app/routes/individual_response.py

```diff
@@ -96,8 +104,13 @@ def request_individual_response(schema, questionnaire_store):
 @individual_response_blueprint.route("/<list_item_id>/how", methods=["GET", "POST"])
 @with_questionnaire_store
 @with_schema
-def individual_response_how(schema, questionnaire_store, list_item_id):
-    language_code = get_session_store().session_data.language_code
+def individual_response_how(
+    schema: QuestionnaireSchema,
+    questionnaire_store: QuestionnaireStore,
+    list_item_id: str,
+) -> Response | str:
+    # Type ignore: @login_required via Blueprint & @with_schema used together guarantee that sessions_store and session_data are not None
+    language_code: str | None = get_session_store().session_data.language_code  # type: ignore
```

Comment:
You should be able to use the `get_locale` method.

Reply:
Ya, it should work fine, `get_locale` used to just return the language from session store, but we recently started setting that language in the cookie session, so it should end up with the same value as if it were from session store.

Reply:
Made use of get_locale().language throughout, and type hinted this to be str, not str | None as it is defaulted if not set 👍